src/main/resources/english.yml

```diff
@@ -1,5 +1,5 @@
 name: SiegeWar
-version: 0.05
+version: 0.06

```

Comment:
Doesn't require a language version bump, 0.0.5 is already an un-released version so files will be updated with the changes you've made below.